### PR TITLE
Complete Pirates of Sartosa

### PR DESCRIPTION
**Pull request by Mjb141**

### CODE REVIEW

**Comment by Mjb141** [comment]
@msolefonte This is completed and needs reviewing.

File: src/script/vco-ie/main_warhammer/wh2_dlc11_cst_pirates_of_sartosa/missions.lua [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		{
+			victory_type vco_victory_type_cst_sar_3_dead_man_s_vengeance;
+			key wh_main_short_victory;
+			issuer CLAN_ELDERS;
+			primary_objectives_and_payload
+			{
+			 
+				objective
+				{
+					override_text mission_text_text_vco_cst_sar_3_dead_man_s_vengeance;
```
Comment (msolefonte) [inline src/script/vco-ie/main_warhammer/wh2_dlc11_cst_pirates_of_sartosa/missions.lua:119]: I would merge this text with the CONTROL_N_REGIONS_INCLUDING one.

**Comment by msolefonte** [comment]
Has this all been tested locally? Mostly rewards

File: src/script/vco-ie/main_warhammer/wh2_dlc11_cst_pirates_of_sartosa/missions.lua (hunk 1 above)
Comment (Mjb141) [reply src/script/vco-ie/main_warhammer/wh2_dlc11_cst_pirates_of_sartosa/missions.lua:136]: I actually prefer it separated, I find it clearer. How attached to this are you?

**Comment by Mjb141** [comment]
> Has this all been tested locally? Mostly rewards

I consider this faction finished at this point so if there's no more feedback I'll play their campaign and test stuff, or @OhManTFE can do it faster?

File: src/script/vco-ie/main_warhammer/wh2_dlc11_cst_pirates_of_sartosa/missions.lua [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+					type CONSTRUCT_N_BUILDINGS_INCLUDING;
+					faction wh2_dlc11_cst_pirates_of_sartosa;
+					total 1;
+
+					building_level wh2_dlc11_special_dragon_tooth_lighthouse_1;
+				}
+			
+				objective
+				{
+					type RAZE_OR_SACK_N_DIFFERENT_SETTLEMENTS_INCLUDING;
```
Comment (msolefonte) [inline src/script/vco-ie/main_warhammer/wh2_dlc11_cst_pirates_of_sartosa/missions.lua:75]: Set it back to the Norsca subculture one and hopes and prays CA fixes that in time.

File: src/db/effects_tables_vco.tsv [hunk 3]
```diff
@@ -26,3 +26,4 @@ vco_skv_clan_moulder_strength	weapon_damage.png	461	weapon_damage.png	campaign	t
 vco_cst_zombie_deckhands_missile_strength	military.png	101	military.png	campaign	true
```
Comment (msolefonte) [inline src/db/effects_tables_vco.tsv:26]: There are icons for missile strength and upkeep. Use them here

File: src/db/vco_mission_text.loc.tsv [hunk 4]
```diff
@@ -400,3 +400,6 @@ mission_text_text_vco_ksl_ort_3_wololo	The Old World is rife with corruption, bl
 mission_text_text_vco_skv_mdr_1_yes_yes_prepare_the_lab	Throt is busy deep down in Hell Pit, mutating any beast or Skaven that gets too close to the Flesh Lab! Unlock all Infantry Augments and Monster Augments.	true
 mission_text_text_vco_skv_mdr_2_the_vermintide_comes	All manner of Throt's modified creations have been loosed upon the world, thundering towards Kislev and the South on feet most foul. The Vermintide can't be stopped now, no-never!	true
 mission_text_text_vco_skv_mdr_3_the_frozen_dead	Norscan bodies are perfect for experiments... They're warped, tainted-foul, and with beautiful flesh! What marvelous experiments it could power. Destroy the Norse dwarfs and corrupted man-things, occupy their frozen lands and use their corpses to grow the most powerful mutagens:	true
+mission_text_text_vco_cst_sar_1_captain_of_the_swordfysh	Commander of the Swordfysh, a mighty galleon crewed by hundreds, Aranessa and her crew prefer boarding action to thundering broadsides. The ship roves the Tilean Sea and beyond, raiding all it sees. Make them all fear the crossed tridents upon its sails. 	true
```
Comment (msolefonte) [inline src/db/vco_mission_text.loc.tsv:403]: Add the `<dummy>` tag for these three.

**Comment by msolefonte** [comment]
Are tests done now?

**Comment by Mjb141** [comment]
> Are tests done now?

All except the norsca subculture objective which is impossible to test currently.

**Review by msolefonte — CHANGES_REQUESTED** [review]
Campaign has to be marked as easy. Apart from that everything seems fine.

**Review by msolefonte — APPROVED** [review]
Looks good to me!

**Pull request merged**